src/main/java/org/jongo/RxJongo.java

```diff
@@ -0,0 +1,46 @@
+package org.jongo;
```

Comment:
Ok that looks like a better place for it to live. There appears to be no maven artifact for the extras library though... But I will leave the functional interface in here and then move the Rx implementation to the extras project.